.github/workflows/release.yml

```diff
@@ -31,12 +32,26 @@ jobs:
       - name: 👷 Build packages
         run: pnpm build-all
 
-      - name: ⛴️ Version and Publish
+      - name: 🤖 Configure Git Bot
         run: |
           git config user.name "Release Bot[bot]"
           git config user.email "bot@example.com"
+
+      - name: ⛴️ Version and Publish
+        if: ${{ github.ref == 'refs/heads/main' }}
+        run: |
           npx changeset version
           npx changeset publish
-          git add .
+          git add --all
           git commit --amend --no-edit
           git push --follow-tags
+
+      - name: 📸 Snapshot and Publish
```

Comment:
Whats the purpose of the snapshot releases?
Why not just releasing an alpha version after merged on main?

I think releasing every PR will lead to a lot of pretty much unused versions on npm

Reply:
It's useful to be able to actually install and smoke test the would-be package of an PR before its actually merged. It also ensures that there areWe missed this feature in CoreUI quite a few times. Is there an issue with having many snapshot versions on npm?
If there is we could also remove the PR trigger

Reply:
I don‘t think there is an issue with npm, personally I just don‘t like having hundreds of snapshot versions :)

But its nice to have the option. What do you think about changing it to a manual trigger instead and remove the PR trigger?

Reply:
done

Reply:
You‘d need the workflow_dispatch trigger in order to be able to run the workflow manually:)